packages/appstore/src/components/platform-launcher/platform-launcher.scss

```diff
@@ -1,10 +1,16 @@
-.platform-launcher {
```

Comment:
do not change the css styles here, you can customize it in the file you need to use it

Reply:
@mahdiyeh-fs  I don't understand this.